melodic/distribution.yaml

```diff
@@ -2069,6 +2069,21 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  cyglidar_d1:
+    doc:
+      type: git
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.1
```

Comment:
I'll also suggest that you don't use `v0.1.1` as the branch name.  While this is technically valid, it will be confusing later on when you have tags like `0.1.1`.  I'll suggest using the `main` branch as the branch where possible.